roseau/load_flow/exceptions.py

```diff
@@ -35,12 +35,19 @@ class RoseauLoadFlowExceptionCode(StrEnum):
     BAD_Y_SHUNT_SHAPE = auto()
     BAD_LINE_MODEL = auto()
     BAD_LINE_TYPE = auto()
-    BAD_CONDUCTOR_TYPE = auto()
+    BAD_MATERIAL = auto()
     BAD_INSULATOR_TYPE = auto()
```

Comment:
Shouldn't this change as well?
```suggestion
    BAD_INSULATOR = auto()
```